tools/AzureRM/AzureRM.psd1

```diff
@@ -244,8 +241,8 @@ PrivateData = @{
         - Providing invalid AUDIT_CHANGED_GROUP value for auditing cmdlets no longer throws an error and will be removed in an upcoming release.
     - Fixed issue https://github.com/Azure/azure-powershell/issues/5046
         - AuditAction parameter in auditing cmdlets is no longer being ignored
-    - Fixed an issue in Auditing cmdlets when ''Secondary'' StorageKeyType is provided
-        - When setting blob auditing, the primary storage account key was used instead of the secondary key when providing ''Secondary'' value for StorageKeyType parameter.
+    - Fixed an issue in Auditing cmdlets when 'Secondary' StorageKeyType is provided
+        - When setting blob auditing, the primary storage account key was used instead of the secondary key when providing 'Secondary' value for StorageKeyType parameter.
     - Changing the wording for confirmation message from Set-AzureRmSqlServerTransparentDataEncryptionProtector
```

Comment:
do not modify any tools\ files

Reply:
ok; it's not intended, probably by some auto script.